src/js/components/SelectMultiple/SelectionSummary.js

```diff
@@ -131,7 +146,17 @@ const SelectionSummary = ({
                 ? `Select all ${options.length} options`
                 : `${value?.length} options selected. Clear all?`
             }
-            label={showSelectAll ? 'Select All' : 'Clear All'}
+            label={
+              showSelectAll
+                ? format({
+                    id: 'selectMultiple.selectAll',
+                    messages,
+                  })
+                : format({
+                    id: 'selectMultiple.clearAll',
+                    messages,
+                  })
+            }
```

Comment:
Can avoid that much duplication:
```suggestion
            label={format({
              id: showSelectAll ? 'selectMultiple.selectAll' : 'selectMultiple.clearAll',
              messages,
            })}
```